### PR TITLE
Template names & cleanups

### PR DESCRIPTION
**Pull request by torotil**
- Rename template files so that they allow follow the `theme-hook-name.tpl.php` naming convention.
- A bit more renaming in campaignion_bar.
- A bit of unused code removal.
- Update for travis.

### CODE REVIEW

File: campaignion_bar/theme/campaignion-bar.tpl.php [hunk 1]
```diff
@@ -24,35 +24,4 @@
 
 <div id="ae-widepopups">
   <?php print theme('campaignion_bar_menu', array('widebar' => TRUE, 'links' => menu_tree_all_data('ae-menu'), 'wide' => array('New'))); ?>
```
Comment (katzmo) [inline campaignion_bar/theme/campaignion-bar.tpl.php:26]: This is still `ae-menu`?

**Review by katzmo — COMMENTED** [review]
Have the removed markup and styles never been used?

File: campaignion_bar/theme/campaignion-bar.tpl.php (hunk 1 above)
Comment (torotil) [reply campaignion_bar/theme/campaignion-bar.tpl.php:26]: Yes, I didn’t touch the menu name (or the styles).

**Comment by torotil** [comment]
> Have the removed markup and styles never been used?

I have done a `ack hotkey-` over the code-base and that came up empty except for the code I’ve removed. The reason I looked more closely is because the template for this markup still has an advocacy engine link in it.

**Review by katzmo — APPROVED** [review]

**Pull request merged**